webknossos/webknossos/client/_defaults.py

```diff
@@ -1 +1,2 @@
 DEFAULT_WEBKNOSSOS_URL = "https://webknossos.org"
+DEFAULT_WEBKNOSSOS_DATASTORE_URL = "https://data-humerus.webknossos.org"
```

Comment:
PS: Is this used for something?

Reply:
No, just a merge artifact ( I had originally started out building something like #497 myself ). Good catch!